content/en/docs/developer-documentation/build-install-rpm.md

```diff
@@ -68,11 +72,16 @@ systemctl enable crio --now
 systemctl enable microshift --now
 ```
 
+To install OpenShift and Kubernetes clients, refer to [Getting Started: Install Clients]({{< ref "/docs/getting-started/_index.md#install-clients" >}}).
+
+To configure the kubeconfig, refer to [Getting Started: Copy or Set Kubeconfig]({{< ref "/docs/getting-started/_index.md#copy-or-set-kubeconfig" >}}).
+
+It is now possible to run `oc` or `kubectl` commands against the MicroShift environment.
+
 Verify that MicroShift is running:
 
 ```sh
-export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig
-sudo oc get pods -A
+oc get pods -A
```

Comment:
Our docs are inconsistent on whether instructions are from the perspective of a non-privileged or root user. IMHO, we should converge on running things as non-privileged user where possible.

It'd still make sense to run `oc` sudo-less. For that, how about replacing the `export` with `sudo install -v -D -b -o $(id -u) -g $(id -g) /var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config`?

Reply:
Ok... don't use that `install` command, it's buggy: `config` gets the right ownership, but in case the `~/.kube` dir doesn't exist, it'll create it with `root` ownership, which leads to some byzantine errors with `oc`. So guess `mkdir -p` + `cp` + `chown` is the right path here.

Reply:
agree with assuming non-privileged user, i'll update that

Reply:
updated